mkt/constants/payments.py

```diff
@@ -97,6 +97,7 @@
 )
 
 PROVIDER_LOOKUP = dict([(v, k) for k, v in PROVIDER_CHOICES])
+PROVIDER_LOOKUP_INVERTED = dict([(k, v) for k, v in PROVIDER_CHOICES])
```

Comment:
I guess you are right, I looked at the one above and wanted the opposite. Its actually just `dict(PROVIDER_CHOICES)` as well.